### ISSUE TITLE
TF Apply failures - error 409 already exists

### ISSUE THREAD

**Issue by rchapman83** [terraform-google-modules/terraform-example-foundation#971]
### TL;DR

Unable to recover TF Apply post hitting the project quota limit. Error 409: Resource already exists.  

### Terraform Configuration

```hcl
Section 1-org
```

### Terraform Version

```sh
Terraform v1.4.6

- Installed hashicorp/google-beta v4.64.0 (signed by HashiCorp)
- Installed hashicorp/random v3.5.1 (signed by HashiCorp)
- Installed hashicorp/time v0.9.1 (signed by HashiCorp)
- Installed hashicorp/null v3.2.1 (signed by HashiCorp)
```

### Additional information

Hi,
I am newish to Terraform and attempting to use the terraform-example-foundation repo to create a secure baseline landing zone for our organisation. 

Everything was running smoothly until I hit a project quota limit I was unaware of. I believe this is remedied however, I am unable to re-run the terraform plan + apply via Cloud Build as it halts when it discovers resources that already exist (Error 409) during the apply step.

I've tried using taint and replace as advised on various forums however, these do not seem to resolve these issues. Perhaps because I'm using them incorrectly, I'm unsure.  Any help in overcoming this issue would be greatly appreciated. 

Example error from build log - 

│ Error: Error creating Contact: googleapi: Error 409: There is already a contact with the given email address grp-gcp-billing-admins@XXXXX.com. To add new category subscriptions, please use UpdateContact instead.
│ Details:
│ [
│   {
│     "@type": "type.googleapis.com/google.rpc.ErrorInfo",
│     "domain": "essentialcontacts.googleapis.com",
│     "reason": "CONTACT_ALREADY_EXISTS"
│   }

│ Error: Error creating folder 'fldr-common' in 'organizations/50XXXXXXXX': Error waiting for creating folder: Error code 9, message: Folder reservation failed for parent [organizations/50XXXXXXXXX], folder [] due to constraint: The folder operation violates display name uniqueness within the parent.
│ 
│   with google_folder.common,
│   on folders.tf line 21, in resource "google_folder" "common":
│   21: resource "google_folder" "common" {

**Comment by bharathkkb** [comment]
@rchapman83 You can try to [import](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_folder#import) the folder into your state or if the folder does not contain anything, you can delete it via gcloud/UI and let TF recreate it.
